src/test/java/com/based/basedsurvey/controller/ResultsSurveyControllerTest.java

```diff
@@ -38,7 +43,7 @@ public void testEmptyQuestionResults() throws Exception{
         String name = "survey1";
         Survey s = new Survey(name);
         surveyRepository.save(s);
-        mockMvc.perform(get("/survey/" + s.getId() + "/results"))
+        mockMvc.perform(get("/survey/1/results"))
```

Comment:
out of curiosity, why hard code the survey ID instead of using getId()?

Reply:
[it follows industry standard for testing](https://tinyurl.com/industry-standards-for-java)
https://tinyurl.com/industry-standards-for-java